Documentation/project-docs/standard-platform.md

```diff
@@ -23,9 +23,9 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
-- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
+- **.NET Platform Standard** - a specific versioned set of reference assemblies that all .NET Platforms must support as defined in the [CoreFX repo](https://github.com/dotnet/corefx).
 - **PCL** - Portable Class Library
-- **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1
+- **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1, MonoTouch, UWP, etc.
```

Comment:
Is .NET Framework 4.5.1 considered a different platform from .NET Framework 4.5?  If so, I'd add that as well, to help highlight how versions relate, e.g. are .NET Framework 4.5 and .NET Framework 4.5.1 two different versions of the same platform, or are they two different platforms?